### PR TITLE
Port get buffer

### PR DESCRIPTION
**Pull request by shanavas786**

### CODE REVIEW

File: rust_src/src/buffers.rs [hunk 1]
```diff
@@ -221,3 +223,33 @@ pub fn overlayp(object: LispObject) -> LispObject {
...
+/// Like Fassoc, but use Fstring_equal to compare
+/// (which ignores text properties), and don't ever quit.
+fn assoc_ignore_text_properties(key: LispObject, list: LispObject) -> LispObject {
+    let result = list.iter_tails_safe().find(|&item| {
+        string_equal(car(item.car()), key).is_not_nil()
+    });
+    if let Some(elt) = result {
+        elt.car()
+    } else {
+        Qnil
```
Comment (birkenfeld) [inline rust_src/src/buffers.rs]: Use `LispObject::constant_nil()` instead.

**Comment by shanavas786** [comment]
conflicts resolved

**Comment by shanavas786** [comment]
rebased

**Comment by Wilfred** [comment]
Thank you :)

**Pull request merged**